conf/test.config

```diff
@@ -20,21 +20,12 @@ params {
     max_time   = 6.h
 
     // Input data
-    //   input = 'https://raw.githubusercontent.com/nf-core/test-datasets/smrnaseq/testdata/samplesheet.csv'
-    // Genome references
- //   bt_indices = [
- //     'https://github.com/nf-core/test-datasets/raw/smrnaseq/reference'
- //   ]
- //   gtf = 'https://github.com/nf-core/test-datasets/raw/smrnaseq/reference/genes.gtf'
- //   mature = 'https://github.com/nf-core/test-datasets/raw/smrnaseq/reference/mature.fa'
- //   hairpin = 'https://github.com/nf-core/test-datasets/raw/smrnaseq/reference/hairpin.fa'
- //   mirna_gtf = 'https://github.com/nf-core/test-datasets/raw/smrnaseq/reference/hsa.gff3'
 
-    input = 'https://github.com/nf-core/test-datasets/raw/smrnaseq-better-input/testdata/samplesheet.csv'
-    fasta = 'https://github.com/nf-core/test-datasets/raw/smrnaseq-better-input/reference/genome.fa'
-    mature = 'https://github.com/nf-core/test-datasets/raw/smrnaseq-better-input/reference/mature.fa'
-    hairpin = 'https://github.com/nf-core/test-datasets/raw/smrnaseq-better-input/reference/hairpin.fa'
-    mirna_gtf = 'https://github.com/nf-core/test-datasets/raw/smrnaseq-better-input/reference/hsa.gff3'
+    input = 'https://github.com/nf-core/test-datasets/raw/smrnaseq/testdata/samplesheet.csv'
```

Comment:
Maybe we could implement a similar versioning such as in the rnaseq pipeline e.g. https://github.com/nf-core/test-datasets/tree/rnaseq/samplesheet/v3.4

Reply:
Great idea, especially if some potential future changes are coming in that might have to adjust the format at some point again 👍🏻

Reply:
ok, should I merge and then coordinate this? or do you want to do it now?